src/System.Private.CoreLib/shared/System/TimeZoneInfo.cs

```diff
@@ -1989,5 +1979,88 @@ private static void ValidateTimeZoneInfo(string id, TimeSpan baseUtcOffset, Adju
                 }
             }
         }
+
+        private static readonly TimeSpan MaxOffset = TimeSpan.FromHours(14.0);
```

Comment:
>MaxOffset [](start = 41, length = 9)

why you didn't use the numbers direct (e.g. +/- 14) instead of defining these statics? with static would be slower.

Reply:
I didn't want to do subtraction on doubles (it will use long internally now) - if we decide to remove Normalize method then I'll convert back